Source/Core/Core/IOS/Network/WD/Command.cpp

```diff
@@ -60,14 +120,15 @@ IPCCommandResult NetWDCommand::IOCtlV(const IOCtlVRequest& request)
 
   case IOCTLV_WD_GET_INFO:
   {
-    Info* info = (Info*)Memory::GetPointer(request.io_vectors.at(0).address);
-    memset(info, 0, sizeof(Info));
-    // Probably used to disallow certain channels?
-    memcpy(info->country, "US", 2);
-    info->ntr_allowed_channels = Common::swap16(0xfffe);
-
-    const Common::MACAddress address = IOS::Net::GetMACAddress();
-    std::copy(address.begin(), address.end(), info->mac);
+    // Nintendo does not actually check this...
+    if (request.in_vectors.size() + request.io_vectors.size() < 1)
```

Comment:
Would it be clearer to do `request.in_vectors.empty()` ?

Reply:
I think writing it like this makes it clearer that the code is checking for the total number of vectors, and that it requires the request to have at least one vector (in or in-out). This could be turned into !in.empty() && !io.empty() but then it would be inconsistent with the other ioctl handlers which don't necessarily only receive one vector.

Reply:
Gotcha.  I missed that they were separate variables